add a voucher_subscriber option (#15)
grafeas/voucher: README.md

Note that `provenance` and the dynamic checks require the prescence of build metadata in your metadata store. While unsigned metadata is valid, to ensure that you are trusting metadata that hasn't been forged, it is recommended that you use signed metadata as well.

## Voucher Server, Subscriber, and Client

This repository contains three tools:

> - [Voucher server](cmd/voucher_server/README.md): intended to run in your infrastructure to respond to CI/CD pipeline requests.

Reviewer: ```suggestion
- [Voucher Server](cmd/voucher_server/README.md): intended to run in your infrastructure to respond to CI/CD pipeline requests.
```

> - [Voucher subscriber](cmd/voucher_subscriber/README.md): intended to listen for pub/sub messages for image creations and automatically vouch images.

Reviewer: ```suggestion
- [Voucher Subscriber](cmd/voucher_subscriber/README.md): a server that accepts image creation pub/sub messages from GCR, and automatically vouches the images referenced in those messages.
```

> - [Voucher subscriber](cmd/voucher_subscriber/README.md): intended to listen for pub/sub messages for image creations and automatically vouch images.

Reviewer 2: Just to confirm, Voucher Subscriber does not respond to client requests, correct?

Reviewer: This is correct! We should probably clarify that in the README

> - [Voucher client](cmd/voucher_client/README.md): an example of a Voucher API client that you can use directly in your CI/CD pipeline or as a basis for your own code.

Reviewer: ```suggestion
- [Voucher Client](cmd/voucher_client/README.md): an example of a Voucher API client that you can use directly in your CI/CD pipeline or as a basis for your own code.
```

> - [Voucher client](cmd/voucher_client/README.md): an example of a Voucher API client that you can use directly in your CI/CD pipeline or as a basis for your own code.

Reviewer 2: Maybe worth mentioning that the client sends requests to the Voucher Server, since there are three entities now.
